tests/genomic_file/test_genomic_file_resources.py

```diff
@@ -156,17 +106,18 @@ def test_new_indexd_error(client, entities):
         'data_type': 'Aligned Reads',
         'file_format': 'bam',
         'urls': ['s3://bucket/key'],
-        'hashes': {'md5': 'd418219b883fce3a085b1b7f38b01e37'},
-        'sequencing_experiment_id': 'SE_AAAAAAAA',
+        # 'hashes': {'md5': 'd418219b883fce3a085b1b7f38b01e37'},
```

Comment:
^

Reply:
Meant to delete that line. This test is supposed to ensure that when there is a problem with the input data, the post should fail and indexd should not create a file. Hashes is a required field so without it present the post should fail as expected